schemas/edges/is_consumed_by.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@JamesJeffryes My only feeback on this PR is that "consumes" is an odd word choice. To me, it means to "use up". My leaning would be something like "type_is_used_in_method" or similar.

Reply:
I chose "is_consumed_by" to pair with "is_produced_by" because I find the super long edge names clunky. I suppose I could do "is_input/output_in"

Reply:
Another thing I was thinking that might feel clunky/annoying now, but might be important if we later scale up to a 100 collections, is to make use of prefixes on collection names as a simple namespacing mechanism. For example, all the stuff related workspace data could have the "ws_*" prefix.